README.md

```diff
@@ -17,12 +17,6 @@ Follow the instructions here: [https://github.com/bocoup/threeflows#local-databa
 
 This is the list of all environment variables that should be exported prior to running the Teacher Moments server in the command line:
 ```
-export PGUSER=
-export PGPASSWORD=
-export PGDATABASE=
-export PGHOST=
-export PGPORT=
```

Comment:
Do we still need to export these elsewhere? Or are they covered in database.json?